readthedocs/builds/version_slug.py

```diff
@@ -34,9 +34,8 @@
 
 
 class VersionSlugField(models.CharField):
-    """
-    Implementation inspired by ``django_extensions.db.fields.AutoSlugField``.
-    """
+
+    """Inspired by ``django_extensions.db.fields.AutoSlugField``."""
```

Comment:
The pep257 plugin complains that the original should be pulled into a one-line docstring, but if you do, you exceed the 79-character line limit, so I tweaked the wording to fit.